hooks/usePolybase.tsx

```diff
@@ -1,13 +1,13 @@
-import { Polybase } from "@polybase/client";
+import { Polybase, Signer } from "@polybase/client";
 import { Address } from "wagmi";
-const usePolybase = () => {
+const usePolybase = (signer:Signer) => {
```

Comment:
```suggestion
const usePolybase = (signer?: Signer) => {
```
This way you don't need to pass a signer if you do not want to... so you can revert changes from earlier which added a signer which didnt have before